packages/tools/client-debugger/client-debugger-chrome-extension/src/react-components/TelemetryView.tsx

```diff
@@ -26,7 +25,7 @@ const loggingContext = "EXTENSION(DebuggerPanel:Telemetry)";
  */
 export function TelemetryView(): React.ReactElement {
 	const messageRelay = useMessageRelay();
-
+	const [pageLimit, setPageLimit] = React.useState<number>(100);
```

Comment:
```suggestion
	const [maxEventsToDisplay, setMaxEventsToDisplay] = React.useState<number>(100);
```

Reply:
I would also recommend extracting `100` as a constant somewhere so we can document it.